ddtrace/contrib/flask/middleware.py

```diff
@@ -116,7 +116,10 @@ def _finish_span(self, response=None, exception=None):
                     span.set_tag(errors.ERROR_TYPE, type(exception))
                     span.set_tag(errors.ERROR_MSG, exception)
 
-                span.resource = compat.to_unicode(request.endpoint or '').lower()
+                # the endpoint that matched the request is None if an exception
+                # happened so we fallback to a common resource
+                resource = code if not request.endpoint else request.endpoint
```

Comment:
@LotharSee I'm using the ``code`` ([here][1]) that could be ``None`` in some really edge cases. The case is:
* we don't have a response
* but this is not an exception

Idea of providing a better name than ``none``?

[1]: https://github.com/DataDog/dd-trace-py/pull/151/files#diff-ec4c4d8c9993c49cadac94e886db2c55R109